target/linux/mvebu/image/cortexa53.mk

```diff
@@ -30,6 +30,16 @@ define Device/globalscale_espressobin-emmc
 endef
 TARGET_DEVICES += globalscale_espressobin-emmc
 
+define Device/globalscale_espressobin-ultra
+  $(call Device/Default-arm64)
+  DEVICE_VENDOR := Marvell
+  DEVICE_MODEL := ESPRESSObin
+  DEVICE_VARIANT := Ultra
+  DEVICE_PACKAGES += kmod-i2c-pxa kmod-rtc-pcf8563
+  SOC := armada-3720
```

Comment:
with #3405 merged you can now add
`BOOT_SCRIPT := espressobin`

Reply:
added in the last change, also updated uboot-envtools as discussed earlier.